specification/v3.6/OSDM-online-api-v3.6.0.yml

```diff
@@ -712,6 +715,107 @@ paths:
           $ref: '#/components/responses/ServiceUnavailableResponse'
         'default':
           $ref: '#/components/responses/DefaultErrorResponse'
+
+  /offer-overview-leg:
```

Comment:
/offer-overview-leg

This operation is intended to be used for a kind of calendar view.
In my opinion, "leg" is not the best wording. The request specifies only the origin and destination.
Connections between origin and destination can involve intermediate stops and multiple legs.

IMO Better woring would be /offer-overview-route, since the request specifies only the origin and destination, and the term "leg" implies a single segment, which may not reflect the potential complexity of the journey.